.github/workflows/ansible-lint.yml

```diff
@@ -0,0 +1,20 @@
+name: ansible-lint
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Ansible Lint # Naming the build is important to use it as a status check
```

Comment:
Can you remove all the comments imported from the [ansible-lint-action](https://github.com/ansible/ansible-lint-action) example from in this file?